wormhole-connect/src/views/v2/Bridge/ReviewTransaction/WalletBalanceWarning.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import AlertBanner from 'components/v2/AlertBanner';
+import { Stack, Typography, useTheme } from '@mui/material';
+import { toFixedDecimals } from 'utils/balance';
+
+export type WalletBalanceWarningProps = {
+  isCheckingBalance: boolean;
+  isBalanceEnough: boolean;
+  walletBalance?: number;
+  networkCost?: number;
+  symbol?: string;
+};
+
+export default function WalletBalanceWarning({
+  isCheckingBalance,
+  isBalanceEnough,
+  walletBalance,
+  networkCost,
+  symbol,
+}: WalletBalanceWarningProps) {
+  const theme = useTheme();
+  const content = isCheckingBalance
+    ? `Checking if wallet balance is enough...`
+    : `Your wallet balance is not enough to cover the network cost. Please add more funds to your wallet.`;
```

Comment:
"Insufficient token balance to cover the network cost. Please add more funds to your wallet." ?